src/common/theme/components/ModalTheme/ModalTheme.tsx

```diff
@@ -0,0 +1,17 @@
+import { modalAnatomy as parts } from '@chakra-ui/anatomy';
+import { createMultiStyleConfigHelpers } from '@chakra-ui/styled-system';
+
+import { baseStyle } from './baseStyle';
+
+const { defineMultiStyleConfig } = createMultiStyleConfigHelpers(parts.keys);
+
+type defaultProps = ReturnType<typeof defineMultiStyleConfig>['defaultProps'];
+
+const modalDefaultProps = {
+  isCentered: true,
+} as defaultProps; // type casting is necessary because defaultProps is not accepting ModalProps
+
+export const ModalTheme = defineMultiStyleConfig({
+  baseStyle,
```

Comment:
why not put the config in same file?

Reply:
Cause it was getting long. so had to put it in another file just so it is scaleable.